pages/main_page.py

```diff
@@ -0,0 +1,13 @@
+from pages.base_page import BasePage
+from pages.locators import MainPageLocators
+
+
+class MainPage(BasePage):
+    def open_page(self):
+        self.browser.get(MainPageLocators.URL)
```

Comment:
В будущем мы не знаем какой `url` нам нужно будет передавать, но уже сейчас метод `open_page()` может открыть только единственный `url`. Хорошая практика передавать `url` из тестовой функции, либо при создании экземпляра класса.
```suggestion
    def open_page(self, url):
        self.browser.get(url)
```